lib/log_analyzer/analyzer.rb

```diff
@@ -36,6 +36,8 @@ def order(by: :time)
         @stats = @stats.sort{|a, b| a[0] <=> b[0] }
       when :time
         @stats = @stats.sort{|a, b| a[1].avg <=> b[1].avg }
+      when :rtime
+        @stats = @stats.sort{|a, b| a[1].avg <=> b[1].avg }.reverse
```

Comment:
I don't like idea of .reverse
better just to swap elements

`@stats = @stats.sort{|a, b| b[1].avg <=> a[1].avg }`

please change and I'll merge

Reply:
Hi, thank you for your suggestion, I change it